packager/react-packager/src/DependencyResolver/Module.js

```diff
@@ -66,23 +67,31 @@ class Module {
   }
 
   getDependencies() {
-    return this._read().then(data => data.dependencies);
+    return this.read().then(data => data.dependencies);
   }
 
   getAsyncDependencies() {
-    return this._read().then(data => data.asyncDependencies);
+    return this.read().then(data => data.asyncDependencies);
   }
 
   invalidate() {
     this._cache.invalidate(this.path);
   }
 
-  _read() {
+  read() {
```

Comment:
@vjeux just pointing out that I renamed this method as we discussed, as to allow for proper class inheritance and to not break due to `babel-plugin-class-private-props`.

cc @martinbigio